### PR TITLE
Update dependencies.

### PR DESCRIPTION
**Pull request by JimiC**
Should be merged right before next release.

### CODE REVIEW

File: src/commands/index.ts [hunk 1]
```diff
@@ -226,6 +228,8 @@ function handleAction(btn: models.IVSCodeMessageItem, callback?: (...args: any[]
         handlePreset();
       }
       break;
+    default:
+      break;
```
Comment (jens1o) [inline src/commands/index.ts]: Wut?

Comment (JimiC) [inline src/commands/index.ts]: To make `bitHound` happy (without it, it complains about it with a warning).

Comment (jens1o) [inline src/commands/index.ts]: I mean, why don't we remove that rule?

Comment (JimiC) [inline src/commands/index.ts]: It's not our rule but `bitHound`'s.

Comment (jens1o) [inline src/commands/index.ts]: :(

**Pull request merged**